spec/javascripts/host-content-iframe.spec.js

```diff
@@ -0,0 +1,167 @@
+/* global Turbo */
+
+describe('GOVUK.Modules.HostContentIframe', function () {
+  'use strict'
+
+  let module
+  let instance
+
+  beforeEach(function () {
+    window.Turbo = {
+      visit: jasmine.createSpy('visit')
+    }
+
+    module = createIframe()
+    instance = new window.GOVUK.Modules.HostContentIframe(module)
+    instance.init()
+
+    triggerLoadEvent(module)
+  })
+
+  afterEach(function () {
+    document.body.removeChild(module)
+    delete window.Turbo
+  })
+
+  describe('when clicking a link', function () {
+    let link
+
+    beforeEach(function () {
+      link = module.contentDocument.querySelector('a')
+    })
+
+    it('sets cursor to wait', function () {
+      link.click()
+      expect(link.style.cursor).toBe('wait')
+    })
+
+    it('calls navigateViaTurbo with the link href', function () {
+      link.click()
+      expect(Turbo.visit).toHaveBeenCalledWith('https://example.gov.uk/test', {
+        frame: 'preview_frame_123',
+        action: 'advance'
+      })
+    })
+  })
```

Comment:
Nicely done, can't have been easy to figure out how to get a good UX paging through content in an iFrame... 🙇